qa/L0_backend_python/python_test.py

```diff
@@ -208,6 +201,29 @@ def test_init_args(self):
                 result.as_numpy("OUT") == 7,
                 "Number of keys in the init args is not correct")
 
+    def test_ensemble(self):
+        model_name = "ensemble"
+        shape = [4]
+        with httpclient.InferenceServerClient("localhost:8000") as client:
+            input_data_0 = np.random.random(shape).astype(np.float32)
+            input_data_1 = np.random.random(shape).astype(np.float32)
+            inputs = [
+                httpclient.InferInput("INPUT0", input_data_0.shape,
+                                      np_to_triton_dtype(input_data_0.dtype)),
+                httpclient.InferInput("INPUT1", input_data_1.shape,
+                                      np_to_triton_dtype(input_data_1.dtype))
+            ]
+            inputs[0].set_data_from_numpy(input_data_0)
+            inputs[1].set_data_from_numpy(input_data_1)
+            result = client.infer(model_name, inputs)
+            output0 = result.as_numpy('OUTPUT0')
+            output1 = result.as_numpy('OUTPUT1')
+            self.assertIsNotNone(output0)
```

Comment:
Can you compare output0 and output1 against calculated values using input_data_0 and input_data_1 to make sure you get the expected values back.

Reply:
I did compare it in the lines below.